Documentation/csi_rn_changelog_1.6.0.md

```diff
@@ -0,0 +1,15 @@
+# 1.5.0 \(March 2021\)
+
+IBM® block storage CSI driver 1.5.0 provides a range of enhancements and resolves the following issues:
+
+-   New support for IBM Cloud® Satellite
+-   Added automatic seamless upgrade capabilities
+-   Additional support for Kubernetes 1.20 and Red Hat® OpenShift® 4.7 with x86 architecture
+-   Added Red Hat OpenShift web console installation support for IBM Z® and IBM Power® Systems
+
+|Ticket ID|Severity|Description|
+|---------|--------|-----------|
+|**CSI-TBD**|Moderate|**Fixed:** TBD|
```

Comment:
I think all the `TBD`s can be finished before merging this PR.
[this](https://www.ibm.com/docs/en/blockstg-csi-driver/1.5.0?topic=log-150-march-2021) might help.

Reply:
as we discussed separately, we'll leave it all in one single PR.